chamber/forms/validators.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, content_types):
     def __call__(self, data):
         data.open()
         with magic.Magic(flags=magic.MAGIC_MIME_TYPE) as m:
-            mime_type = m.id_buffer(data.read(1024))
+            mime_type = m.id_buffer(data.read(2048))
```

Comment:
I was getting wrong mime type `application/zip` for a XLSX file. This fixes it an is even recommended in official docs as minimal value. https://github.com/ahupp/python-magic